### PR TITLE
T18806: Build SignalR library x64

### PR DESCRIPTION
**Pull request by azamatbakhov**
Specify library name and version:  **lib/1.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/signalr-qt/all/conanfile.py [hunk 1]
```diff
@@ -55,6 +55,21 @@ def _find_qmake(self):
                 return os.path.join(bin, "qmake")
         return "qmake"
 
+    def _force_target_dll(self, qmake_pro_file):
+        if not os.path.isfile(qmake_pro_file):
+            self.output.info("Failed to set the taget to dll: [%s]" % qmake_pro_file)
```
Comment (Artalus) [inline recipes/signalr-qt/all/conanfile.py:60]: опечатка, `taRget`

File: recipes/signalr-qt/all/conanfile.py [hunk 2]
```diff
@@ -55,6 +55,21 @@ def _find_qmake(self):
...
+            return
+
+        with open(qmake_pro_file, 'r') as file :
+            file_content = file.read()
+
+        file_content = file_content.replace("staticlib", "dll")
```
Comment (Artalus) [inline recipes/signalr-qt/all/conanfile.py:66]: это бы через применение патча делать, а не вслепую заменять все staticlib в файле

File: recipes/signalr-qt/all/conanfile.py [hunk 3]
```diff
@@ -83,6 +98,9 @@ def _getenvpath(var):
                                  'QMAKE_LINK=' + value,
                                  'QMAKE_LINK_SHLIB=' + value]
 
+                signalr_pro_file = os.path.join(self._source_subfolder, "SignalRLibraries/SignalRClient/SignalRClient.pro")
```
[the remark this answers is not in the document]
Comment (Artalus) [reply recipes/signalr-qt/all/conanfile.py:101]: поддерживаю идею с добавлением коммента

**Review by Artalus — CHANGES_REQUESTED** [review]

File: recipes/signalr-qt/all/conanfile.py [hunk 4]
```diff
@@ -39,6 +39,11 @@ def source(self):
 
         tools.patch(base_path=self._source_subfolder, patch_file="patches/fix_build_deps.patch", strip=1)
 
+        # We apply this patch to set the target of this library to dll explicitly.
+        # Yet we can't use qmake *.pro -after "CONFIG += dll" for this will affect for all sub projects
+        # wich are statically linked to SignalRClient.dll.
```
Comment (Artalus) [inline recipes/signalr-qt/all/conanfile.py:44]: `wHich`

**Review by Artalus — CHANGES_REQUESTED** [review]

**Review by Artalus — APPROVED** [review]

**Review by worldemar — APPROVED** [review]

**Pull request merged**